### DISCUSSION TITLE
FLIPs qualification proposal, additional inputs, possibly for quadratic voting

### DISCUSSION THREAD

**Discussion by sekisanchi** [idena-network/idena-desktop#674] [Ideas]
I look at the first block of EP74 ceremony, and amazing 300 people or hybrid bot answered just in 30 sec.  However, FLIPS they submit often includes not preferable FLIPs. I think about it for a while, and what I want to mark those as it is I feel.  In specific, Like/Dislike or Intelligent/Not Intelligent check marks, or 1 to 5 level selection for quadratic votings.  For example,I like this FLIP one out of 5 or I think this FLIP is intelligent 3 out of 5. https://scan.idena.io/block/3322964

**Comment by sekisanchi** [comment]
Another option for Like/Dislike  +5 to -5  slide bar

**Comment by aidenaio** [comment]
moving to discussions
